english/ponder/ponder-about-life.md

```diff
@@ -8,3 +8,4 @@ Do you:
 
 [Count the marshmallows?](../count-the-marshmellows/count-the-marshmellows.md)
 
+[Seek s'more options?] (../seek-smore-options/seek-smore-options.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.